vsintegration/tests/UnitTests/LegacyProjectSystem/Tests.ProjectSystem.Configs.fs

```diff
@@ -23,13 +23,10 @@ open UnitTests.TestLib.Utils.FilesystemHelpers
 open UnitTests.TestLib.ProjectSystem
 
 
-[<SetUpFixture>]
```

Comment:
From where is this being called now?
This is a per-namespace init, before any of the tests are run (even when just a handful are being run), right?

Reply:
Thanks for pointing out, this can actually be removed, I forgot to clean this up.

We only need this assembly resolver in one suite, elsewhere no, it was probably a copypaste or necessary with earlier lib versions. 

As in, I tried removing the original (Nunit) fixtures in main, and it didn't affect the test execution and results.